front/src/reducers/osrdconf/stdcmConf/selectors.ts

```diff
@@ -32,6 +32,10 @@ const buildStdcmConfSelectors = () => {
     }
   );
 
+  const getStdcmCompletedSimulations = createSelector([getStdcmSimulations], (simulations) =>
+    simulations.filter((simulation) => simulation.outputs !== undefined)
```

Comment:
Wouldn't it work like that ? 

```suggestion
    simulations.filter((simulation) => simulation.outputs)
```